slices/libpython3.9-stdlib.yaml

```diff
@@ -0,0 +1,392 @@
+package: libpython3.9-stdlib
+
+# The slices in this package have been grouped with inspiration from the Python
+# 3.9 Standard Library (https://docs.python.org/3.9/library/index.html).
+# Aside from the "core" slice which contains the minimal libraries that should
+# come from this package and the "extra" slice which contains miscellaneous
+# libraries, all the other slice definitions are sorted by their names. The
+# "core" slice is placed at the very beginning and the "extra" slice at the
+# very end.
+slices:
+  # The "core" slice provides a very minimal libpython3.9-stdlib
+  core:
+    essential:
+      - libbz2-1.0_libs
+      - libc6_libs
+      - liblzma5_libs
+      - libpython3.9-minimal_libs
+      - mime-support_mime-types
+    contents:
+      # _aix_support and _bootsubprocess are new in 3.9
+      /usr/lib/python3.9/_aix_support.py:
```

Comment:
Odd that this is even included in the deb file since it should never even be imported on a Linux system, but I suppose it's harmless there.

Since [the only time this is imported](https://github.com/python/cpython/blob/865270262ade38b96fce88e32a321caac5332a87/Lib/sysconfig.py#L680-L682) is on an AIX system though, do we want to lump it in with `_osx_support.py` in an `other-os` slice?

Reply:
Actually it looks like this is done as its own slice in Python 3.10 (https://github.com/canonical/chisel-releases/blob/edde443041a628d3b77a9e475a61afd3d5e36e84/slices/libpython3.10-stdlib.yaml#L53-L58), so mimicking that is probably the way to go.